acceptance/lib/helper.rb

```diff
@@ -2,19 +2,28 @@ module Puppet
   module Acceptance
     module Reboot
       # due to a ruby bug the correct error code 1116, is returned modulo 256 = 92
-      SHUTDOWN_ABORT = 'cmd /c shutdown /a'
-      SHUTDOWN_NOT_IN_PROGRESS = 1116 % 256
+      WINDOWS_SHUTDOWN_ABORT = 'cmd /c shutdown /a'
+      WINDOWS_SHUTDOWN_NOT_IN_PROGRESS = 1116 % 256
```

Comment:
Btw, the modulo 256 part is due to a ruby bug, which we worked around in https://tickets.puppetlabs.com/browse/PUP-434. In other words, it should not be necessary to do this when testing on puppet versions >= 3.4.0, though I don't know how easily it is to check the version in the acceptance test when testing open source vs puppet enterprise.